### PR TITLE
REP-251 - Updated seeds for permissions, reworked routes

### PR DESCRIPTION
**Pull request by reserad**
Routes are now separated for user management.

### CODE REVIEW

**Comment by ptorian** [comment]
"Fixed merge issue" needs a better commit message. What does this commit do, why is it necessary?

File: db/repositories/userEmailsRepository.js [hunk 1]
```diff
@@ -94,27 +107,25 @@ class UserEmailsRepository {
                             }
                         });
 
+            // const inserted = emails.map(email => {
```
Comment (badams2527) [inline db/repositories/userEmailsRepository.js]: Please remove the commented code

File: db/repositories/userEmailsRepository.js [hunk 2]
```diff
@@ -94,27 +107,25 @@ class UserEmailsRepository {
...
-
-            await this.uow._models.UserEmail
-                .query(this.uow._transaction)
-                .patch({deleted: true})
-                .whereIn("id", deleted)
-                .returning("*");
-
-            return existingDeletedUserEmails.concat(inserted);
+            
+            // return await this.uow._models.UserEmail
```
Comment (badams2527) [inline db/repositories/userEmailsRepository.js]: Remove here as well

File: api/services/emailService.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class EmailService {
     async sendVerificationEmail(userEmail, uow, request) {
+        console.log(JSON.stringify(userEmail));
```
Comment (ptorian) [inline api/services/emailService.js]: Remove this please

**Pull request merged**